README.md

```diff
@@ -17,8 +17,14 @@ For information on data models and to view the entire DAG check out our dbt [Doc
 ## ✅ How to get started
 
 ### Pre-requisites
-1. You have Medicare CCLF claims data loaded into a data warehouse
-2. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse)
+1. You have Medicare CCLF claims data loaded into a data warehouse.
+2. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse).
+3. This project is dependent on the package `dbt_utils`, include the following in your packages.yml:
+   ```
+   packages:
+     - package: dbt-labs/dbt_utils
+       version: [">=0.9.2"]
```

Comment:
Do you think we should have this open ended version range?  1.0.0 was recently released, and I don't know if they affect us (haven't tested it) but I know there are some breaking changes in it.

In data profiling we have version: [">=0.9.2","<1.0.0"].  I think if we keep what's in the readme we should at least test on 1.0.0 first (maybe you've already done that @sarah-tuva ?)  If so maybe we change DP to be open ended like this one or at least bump up the major to version: [">=0.9.2","<2.0.0"], if not maybe we keep it as <1.0.0 for now?

Reply:
@tuvaforrest I totally agree with keeping the range. I haven't done any testing with 1.0 yet. I just pushed an update to add the range.